conn/spi/spi.go

```diff
@@ -114,23 +128,15 @@ type Conn interface {
 	//
 	// mode specifies the clock and signal polarities, if the bus is using half
 	// duplex (shared MISO and MOSI) or if CS is not needed.
-	DevParams(maxHz int64, mode Mode, bits int) error
-	// TxPackets does multiple operations over the SPI bus.
-	//
-	// The maximum number of bytes can be limited depending on the driver. Query
-	// conn.Limits.MaxTxSize() can be used to determine the limit.
-	//
-	// If the last packet has KeepCS:true, the behavior is undefined. The CS line
-	// will likely not stay asserted. This is a driver limitation.
-	TxPackets(p []Packet) error
+	DevParams(maxHz int64, mode Mode, bits int) (Conn, error)
 }
```

Comment:
I did.

SPI without CS is a bus, SPI with CS is a point-to-point connection.

I implemented a version with a Bus but it felt more clunky to use in practice... It was annoying for the app and the app had to eventually specify a CS anyway, it's not the job of the device driver to chose CS0 or CS1.

I agree DevParams is probably ill-named but it still carries the important meaning about the fact that it is about the communication parameters for the device. Maybe CommParams :/